app/Page_Builder/SiteOrigin_Panels.php

```diff
@@ -41,6 +41,5 @@ public function init() {
 		$this->register_row_layout( 'Row\Wide_Layout' );
 
 		$this->register_widget_layout( '\JustCoded\WP\Framework\Page_Builder\v25\Layouts\Rwd_Widget_Layout', 'Default' );
-		$this->register_widget_layout( 'Widget\Hero_Layout' );
```

Comment:
why you removed this layout? It's an example of some widget layout?

There are no any task to remove it!

Reply:
returned back